config/locales/en.yml

```diff
@@ -2283,11 +2283,19 @@ en:
           primary_was_incarcerated: Were you incarcerated in a county, state, or federal prison for at least 60 days in %{tax_year}?
           spouse_was_incarcerated: Was your spouse incarcerated in a county, state, or federal prison for at least 60 days in %{tax_year}?
           title: You might be eligible for the Increased Excise Tax Credit.
+      az_primary_state_id:
+        az_primary:
+          learn_more_html: |
+            To learn more, visit:<br>
+            <a href="https://azdor.gov/individuals/income-tax-filing-assistance/verifying-identity-and-avoiding-identity-theft" target="_blank">
+            azdor.gov/individuals/income-tax-filing-assistance/verifying-identity-and-avoiding-identity-theft
+            </a>
+          ongoing_effort: In an ongoing effort to protect taxpayers from identity theft, the IRS, state tax agencies, and the tax industry are asking you to provide your driver’s license or state-issued identification number.
```

Comment:
is this something that needs to be implemented/added globally?
https://docs.google.com/document/d/10QhQJuaRQY17tgBOf_um9yKj5UjCHx_rYLlO7wiqPgE/edit?disco=AAABcitsjos

Reply:
I am looking into that. It says different things in Idaho and Arizona final doc so its a bit confusing.

Reply:
Turns out this only for Arizona

Reply:
then we're good here